workers/deployment_manager/.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
```

Comment:
Everything's commented out? Also, does this file work outside of the root?

Reply:
yeah, remnant of your project. will just delete it ;)